app/models/user.rb

```diff
@@ -597,6 +597,22 @@ def unread_high_priority_notifications
     @unread_high_prios ||= unread_notifications_of_priority(high_priority: true)
   end
 
+  def new_personal_messages_notifications_count
+    args = {
+      user_id: self.id,
+      seen_notification_id: self.seen_notification_id,
+      private_message: Notification.types[:private_message]
+    }
+    DB.query_single(<<~SQL, **args).first
```

Comment:
```suggestion
    DB.query_single(<<~SQL, args).first
```

I believe we can just pass the hash?